torch/onnx/symbolic_opset9.py

```diff
@@ -835,24 +835,49 @@ def _prepare_onnx_paddings(dim, pad):
     paddings = paddings[-2::-2] + paddings[-1::-2]
     return paddings
 
+def _convert_padding_node(padding):
+    if sym_help._is_value(padding):
+        if padding.node().kind() == 'onnx::Constant':
+            tval = padding.node()['value']
+            padding = [int(v) for v in tval]
+        elif padding.node().kind() == 'prim::ListConstruct':
+            all_constant = True
+            for v in padding.node().inputs():
+                if v.node().kind() != 'onnx::Constant':
+                    all_constant = False
+                    break
+            if all_constant:
+                padding = [int(v.node()['value']) for v in padding.node().inputs()]
+            else:
+                return sym_help._onnx_opset_unsupported('Pad', 9, 11)
```

Comment:
Thanks, what's the error message in this case? Does it clarify the non-Constant issue?